### ISSUE TITLE
[Accessibility][Quick Search] No announcement when autocomplete suggestions appear

### ISSUE THREAD

**Issue by GrimaceDoesA11y** [adobe/aem-core-wcm-components#2154]
## Bug Report

**Current Behavior**
An auto-complete suggestion list appear after an author-defined number of characters in the search field. There is no announcement for screen reader user when the suggestions appear. 

**Expected behavior/code**
Screen reader users receive audible feedback when the autosuggestion list expands

**Environment**
- AEM version and patch level (e.g. AEM 6.5)
- Core Components version: Quick Search Component v2

**Possible Solution**
Review with internal accessibility team

**Additional context / Screenshots**
Related to: the Quick Search Component 
URL: https://experienceleague.adobe.com/docs/experience-manager-core-components/using/components/quick-search.html?lang=en

**Comment by gabrielwalt** [comment]
I wonder whether this is a duplicate of #1673 - to be verified.

**Comment by GrimaceDoesA11y** [comment]
@gabrielwalt 
What environment or resources can we use to test this in?

**Comment by gabrielwalt** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
This should be fixed with release 2.18 - please make sure to test it with this or a more recent release of the core components. When the autocomplete suggestions appear, the Quick Search component announces how many search results there are when showing those results - it doesn't make sense to announce more than that without confusing the visitor/user.
